styling-components.md

```diff
@@ -0,0 +1,78 @@
+# Component styling
+
+## Hiding elements that have not been upgraded
+Custom Elements that have not been [upgraded](https://developers.google.com/web/fundamentals/web-components/customelements#upgrades) and don't have styles attached can still be rendered by the browser but they likely do not look how they are supposed to. To avoid a *flash of un-styled content* (FOUC) you should be sure to visually hide Custom Elements if they have not been *defined*:
+
+```css
+:not(:defined) {
+    visibility: hidden;
+}
+```
+Note, this will need to be applied by the consuming application and will not be applied by the components themselves.
+
+## Design System
+In FAST, the *design system* is a collection of properties and values that inform the visual design language of the components. These properties are managed and broadcast through implementations of the *DesignSystemProvider*. See the [*DesignSystemProvider* documentation](https://github.com/microsoft/fast-dna/tree/master/packages/web-components/fast-foundation/src/design-system-provider) for more information.
+
+For the purposes of this section, the *DesignSystemProvider* will create CSS Custom Properties that can be consumed by component stylesheets and provide mechanisms to synchronize, update, and consume those properties programmatically.
+
+## Type Ramp
```

Comment:
I'm assuming this is here and subject to content being added; just want to make sure it's not missed :)